### PR TITLE
Pillow fixes

### PR DESCRIPTION
**Pull request by swalladge**
- allow using Pillow 4.0 in dependencies
- fix method call (fromstring hasn't been working since Pillow 3.0)

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Cython>=0.25.2,<0.26
 Twisted>=16.6.0,<16.7
 Jinja2>=2.8,<2.9
-Pillow>=3.4.2,<3.5
+Pillow>=3.4.2,<=4.0
```
Comment (noway421) [inline requirements.txt]: should be starting from 4.0, shouldn't it? You are using new method, so should be new version range

File: feature_server/statusserver.py [hunk 2]
```diff
@@ -116,7 +116,7 @@ def get_overview(self):
         self.last_map_name != self.protocol.map_info.name or
         current_time - self.last_overview > OVERVIEW_UPDATE_INTERVAL):
             overview = self.protocol.map.get_overview(rgba = True)
-            image = Image.fromstring('RGBA', (512, 512), overview)
+            image = Image.frombytes('RGBA', (512, 512), overview)
```
Comment (noway421) [inline feature_server/statusserver.py]: 👍

File: requirements.txt (hunk 1 above)
Comment (swalladge) [inline requirements.txt]: frombytes and the removal of fromstring has been in Pillow since 3.0

Comment (noway421) [inline requirements.txt]: 👍

**Pull request merged**